### PR TITLE
[JENKINS-64107] Convert SSHD module to a plugin

### PR DESCRIPTION
**Pull request by kuisathaverat**
[JENKINS-55582](https://issues.jenkins-ci.org/browse/JENKINS-55582) Convert modules to plugins
[JENKINS-64107](https://issues.jenkins-ci.org/browse/JENKINS-64107) Convert SSHD module to a plugin
[JEP](https://github.com/jenkinsci/jep/pull/326) Convert modules to plugins

* This PR change to use the  sshd-plugin
* Bump the Core to the 2.282

### CODE REVIEW

**Comment by kuisathaverat** [comment]
waiting for the core 2.282 to be released

File: pom.xml [hunk 1]
```diff
@@ -99,9 +99,9 @@
 
         <!-- test dependencies -->
         <dependency>
-            <groupId>org.apache.sshd</groupId>
-            <artifactId>sshd-core</artifactId>
-            <version>2.1.0</version>
+            <groupId>org.jenkins-ci.modules</groupId>
+            <artifactId>sshd</artifactId>
+            <version>3.0.1</version>
```
Comment (kuisathaverat) [inline pom.xml:104]:
```suggestion
            <version>3.0.3</version>
```

**Comment by timja** [comment]
@jeffret-b

**Comment by kuisathaverat** [comment]
a year without progress I close it

**Pull request closed without merging**